src/Editor/Svg/SvgData.tsx

```diff
@@ -0,0 +1,7 @@
+import { GalleryFileInfo } from '../../Gallery/Model';
+
+export type WhpptSvgData = {
+  svgString: string;
+  galleryItemId: string;
+  fileInfo?: GalleryFileInfo;
```

Comment:
Don't think we need the fileInfo here. It lives on the gallery item and we have the gallery item id

Reply:
Done, left only  the `galleryItemId`, for the editor to pass the API.